src/main/scala/com/microsoft/hyperspace/actions/DeleteAction.scala

```diff
@@ -40,4 +43,17 @@ class DeleteAction(final override protected val logManager: IndexLogManager) ext
   }
 
   final override def op(): Unit = { /* Do nothing */ }
+
+  final override protected def event(message: String): HyperspaceEvent = {
+    val sc = SparkSession.getActiveSession.getOrElse {
```

Comment:
I think we need a `trait` like `ActiveSparkSession` to remove duplication. And it sounds reasonable:
```scala
class DeleteAction(...) extends Action with ActiveSparkSession
```

Reply:
thanks @imback82 , added.
Since all Actions have this, is it ok to just change 
`trait Action extends HyperspaceEventLogging with Logging with ActiveSparkSession`

Reply:
Yes, that sounds good.